Structure and content suggestions

supernord opened the pull request:
Some suggestions for content and structure updates for the WorkflowHub `About` documentation pages.

supernord committed:
Update index.md to include onboarding docs

supernord committed:
Change name and update space_team_organisation.md

supernord committed:
Change name and update join_create_teams_spaces.md

fbacall commented on _data/sidebars/documentation.yml:
```diff
@@ -1,18 +1,18 @@
-    - title: Joining a team
-      url: /docs/joining-a-team/
-  - title: Registering a workflow
+    - title: What are Teams, Spaces and Organizations?
+      url: /docs/space_team_organisation/
+    - title: Join or set up Teams & Spaces
+      url: /docs/join_create_teams_spaces/
+  - title: Registering workflows
```
I think use hyphens instead of underscores in filenames to be consistent

fbacall commented on docs/space_team_organisation.md:
```diff
@@ -0,0 +1,63 @@
+---
+title: What are Teams, Spaces and Organizations?
+redirect_from: /Browsing-projects/
+---
+
```
Add a redirect from the old `/what-is-a-team` page

fbacall commented on docs/index.md:
```diff
@@ -1,11 +1,12 @@
+title: Quick start guide
+redirect_from: 
+     - /docs/before-you-start/
+     - /docs/onboarding/
```
Add redirect from `/docs/getting-started`

supernord committed:
New diagram and content for registration overview

supernord committed:
File names updated and redirects added

supernord commented:
Hey @fbacall - thanks for the feedback 😄 
- I updated the file names and added the redirects
- There is also a new figure for the registration overview. What do you think?

fbacall reviewed (approved):
Looks good!
